wgengine/monitor/monitor.go

```diff
@@ -2,27 +2,26 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-// +build linux freebsd
-
 // Package monitor provides facilities for monitoring network
 // interface changes.
 package monitor
 
 import (
+	"sync"
 	"time"
 
 	"tailscale.com/types/logger"
 )
 
-// Message represents a message returned from a connection.
+// message represents a message returned from a connection.
 // TODO(]|[): currently messages are being discarded, so the
 // properties of the message haven't been defined.
-type Message interface{}
+type message interface{}
 
-// Conn represents the connection that is being monitored.
-type Conn interface {
+// platConn represents the connection that is being monitored.
+type platConn interface {
```

Comment:
document that platConn means platformConn? My first two interpretations were real estate plat and platinum, both of which confused me until I mapped correctly :)